packages/components/src/DateTimePickers/pickers/DatePicker/DatePicker.component.js

```diff
@@ -76,75 +40,112 @@ class DatePicker extends React.Component {
-			<div className={theme.container}>
-				<div className={classNames('tc-date-picker-calendar-header', theme['calendar-header'])}>
-					<div className={classNames('tc-date-picker-calendar-row', theme['calendar-row'])}>
+			<table
+				className={theme.container}
+				ref={ref => {
+					this.calendarRef = ref;
+				}}
+			>
+				<caption className="sr-only">TODO: caption, days aria-label, today aria-label</caption>
+				<thead>
+					<tr className={theme['calendar-header']}>
 						{dayNames.map((dayName, i) => (
-							<abbr
-								className={classNames('tc-date-picker-calendar-item', theme['calendar-item'])}
-								key={i}
-								title={dayName}
-							>
-								{dayName.charAt(0)}
-							</abbr>
+							<th scope="col" key={i}>
+								<abbr key={i} title={dayName}>
+									{dayName.charAt(0)}
+								</abbr>
+							</th>
 						))}
-					</div>
-					<hr className={theme.separator} />
-				</div>
-
-				<div className={classNames('tc-date-picker-calendar-body', theme['calendar-body'])}>
+					</tr>
+				</thead>
+				<tbody>
 					{weeks.map((week, i) => (
-						<div
-							className={classNames('tc-date-picker-calendar-row', theme['calendar-row'])}
+						<tr
 							key={i}
+							className={classNames(theme['calendar-row'], 'tc-date-picker-calendar-row')}
 						>
 							{week.map((date, j) => {
-								const isDisabled = this.isDisabledDate(date);
-								const dateNumber = getDate(date);
-
-								return (
-									<div
-										className={classNames('tc-date-picker-calendar-item', theme['calendar-item'])}
-										key={j}
-									>
-										{this.isCurrentMonth(date) && (
-											<DayPickerAction
-												label={dateNumber}
-												isSelected={this.isSelectedDate(date)}
-												isDisabled={isDisabled}
-												isToday={isToday(date)}
-												aria-label={isDisabled ? 'Unselectable date' : `Select '${dateNumber}'`}
+								if (this.isCurrentMonth(date)) {
+									const day = getDate(date);
+									const isDisabled = this.isDisabledDate(date);
+									const isSelected = this.isSelectedDate(date);
+									const shouldBeFocussable =
+										(selectedInCurrentCalendar && isSelected) ||
+										(!selectedInCurrentCalendar && day === 1);
+
+									const className = classNames(
+										theme['calendar-day'],
+										{
+											[theme.selected]: isSelected,
+											[theme.today]: isToday(date),
+										},
+										'tc-date-picker-day',
+									);
+
+									const tdProps = {
+										key: j,
+										className: theme['calendar-col'],
+									};
+									if (isSelected) {
+										tdProps['aria-current'] = 'date';
+									}
+									return (
+										<td {...tdProps}>
+											<button
+												type="button"
+												className={className}
 												onClick={event => {
 													this.props.onSelect(event, date);
 												}}
-											/>
-										)}
-									</div>
-								);
+												disabled={isDisabled}
```

Comment:
isDisabled is a boolean no ? 
You can call it disabled, isDisabled mke me think of a func.

Reply:
Same for selected I think.

Reply:
Changed it

Reply:
still isDisabled :p

Reply:
Are you sure you reviewed the updated code ?
https://github.com/Talend/ui/pull/1827/files#diff-72bc5b2eaefc04f38bafe1a6591d7926R120